ghost-cat-app/src/app/View/register-page/register-page.component.html

```diff
@@ -1,11 +1,39 @@
 <head>
-    <title>Private Register Page</title>
+    <title></title>
     <base href="/">
 </head>
 <body>
+    <h2>Thank you for your interest in GhostCat Analytics!</h2>
+    <p>To register for our services please reach out to PLACEHOLDER@EMAIL.COM</p>
     <div>
         <img class="logo"src="assets/GhostCatLogo.png" alt="Ghost Cat Logo" />
     </div>
-    <h2>Thank you for your interest in GhostCat Analytics!</h2>
-    <p>To register for our services please reach out to PLACEHOLDER@EMAIL.COM</p>
-</body>
\ No newline at end of file
+    <div class="inputs">
```

Comment:
Hey Gage, I think that these inputs actually belong in a different component. It looks like before this pull request the `register-page` component was our landing page for external users, which is separate from the internally facing register page that only ghost cat employees will see. For now, I would create a new placeholder component named something like `internal-register-page`, and I wouldn't make it visible on the website anywhere yet. We still haven't figured out how we're going to do the internal pages yet (Like if it should be a different website on a different domain, or just a hidden feature that only those logged in as ghostcat employees see, etc.), so just a placeholder component is probably all we need.